ldi-orchestrator/ldio-connectors/ldio-amqp/src/test/java/be/vlaanderen/informatievlaanderen/ldes/ldio/AmqpInIntegrationTestSteps.java

```diff
@@ -0,0 +1,135 @@
+package be.vlaanderen.informatievlaanderen.ldes.ldio;
+
+import be.vlaanderen.informatievlaanderen.ldes.ldi.services.ComponentExecutor;
+import be.vlaanderen.informatievlaanderen.ldes.ldi.types.LdiAdapter;
+import be.vlaanderen.informatievlaanderen.ldes.ldio.config.AmqpInConfigKeys;
+import be.vlaanderen.informatievlaanderen.ldes.ldio.config.OrchestratorConfig;
+import be.vlaanderen.informatievlaanderen.ldes.ldio.config.PipelineConfig;
+import be.vlaanderen.informatievlaanderen.ldes.ldio.valueobjects.ComponentProperties;
+import io.cucumber.java.en.And;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import jakarta.jms.*;
+import org.apache.activemq.ActiveMQConnectionFactory;
+import org.apache.jena.rdf.model.Model;
+import org.apache.jena.riot.RDFParser;
+import org.apache.jena.riot.RDFParserBuilder;
+import org.apache.jena.riot.RDFWriter;
+import org.testcontainers.activemq.ArtemisContainer;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Stream;
+
+import static org.apache.jena.riot.RDFLanguages.contentTypeToLang;
+import static org.apache.jena.riot.RDFLanguages.nameToLang;
+import static org.awaitility.Awaitility.await;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class AmqpInIntegrationTestSteps extends AmqpIntegrationTest {
+	public LdioAmqpInRegistrator ldioAmqpInRegistrator = jmsInRegistrator();
+	ArtemisContainer activemq = new ArtemisContainer("apache/activemq-artemis:2.30.0-alpine")
+			.withUser("user")
+			.withPassword("password")
+			.withExposedPorts(61616);
+	private Model inputModel;
+	private String topic;
+	private String contentType;
+	private Map<String, String> config;
+	private List<LdiAdapter.Content> adapterResult;
+	private List<Model> componentExecutorResult;
+	private Session session;
+	private Connection connection;
+	private MessageProducer producer;
+
+	@Given("^I create a topic for my scenario: (.*)$")
+	public void iCreateATopic(String topic) throws JMSException {
+		this.topic = topic;
+		activemq.start();
+
+		ConnectionFactory connectionFactory = new ActiveMQConnectionFactory(activemq.getBrokerUrl());
+		connection = connectionFactory.createConnection(activemq.getUser(), activemq.getPassword());
+		connection.start();
+
+		session = connection.createSession(false, Session.AUTO_ACKNOWLEDGE);
+
+		// Getting the queue
+		Queue queue = session.createQueue(topic);
```

Comment:
this naming is quite confusing as topic != queue in jms

https://activemq.apache.org/how-does-a-queue-compare-to-a-topic